subsys/net/l2/openthread/Kconfig

```diff
@@ -226,6 +226,11 @@ config OPENTHREAD_BORDER_ROUTER
 	help
 	  Enable border router support in OpenThread stack
 
+config OPENTHREAD_COAP
```

Comment:
How is this option used? What happens if you enable it? Perhaps the help text could be elaborated a bit.

Reply:
I've changed it a little bit, hope this is more helpful.